config/crd/bases/management.project-flotta.io_edgedevices.yaml

```diff
@@ -118,6 +118,20 @@ spec:
                         type: integer
                     type: object
                 type: object
+              metricsReceiver:
+                properties:
+                  ? "R\requestNumSamples"
```

Comment:
typo?

Reply:
I changed edgedevice_types but did not execute 'generate'

Reply:
fixed